<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>450</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 45, 45);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(35, 35, 35);
	border-radius:5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../../Commons/resource/asset.qrc">:/icons/asset/database.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Tables</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_minimize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(85, 255, 127);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(85, 255, 127,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_maximize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 170, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 0, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLineEdit" name="add_program">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>120</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Table name</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_36">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>130</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../../Commons/resource/asset.qrc">:/icons/asset/table.svg</pixmap>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_program">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>200</y>
              <width>201</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Create</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../Commons/resource/asset.qrc">
              <normaloff>:/icons/asset/database.svg</normaloff>:/icons/asset/database.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_remove_program">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>200</y>
              <width>211</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Drop</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../Commons/resource/asset.qrc">
              <normaloff>:/icons/asset/minus.svg</normaloff>:/icons/asset/minus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_notification">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>270</y>
              <width>441</width>
              <height>101</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Notification</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QComboBox" name="database_tables">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>291</width>
              <height>38</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>38</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
	color: rgb(255, 255, 255);
	border: 2px solid rgb(45, 45, 45);
	padding-left:10px;
	border-radius: 5px;
}</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_refresh">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>50</y>
              <width>131</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../Commons/resource/asset.qrc">
              <normaloff>:/icons/asset/refresh-cw.svg</normaloff>:/icons/asset/refresh-cw.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Commons/resource/asset.qrc"/>
  <include location="../../../Commons/resource/asset.qrc"/>
 </resources>
 <connections/>
</ui>
